be/src/agent/task_worker_pool.h

```diff
@@ -43,6 +43,8 @@ class ThreadPool;
 
 class TaskWorkerPool {
 public:
+    // You need to modify the content in TYPE_STRING at the same time,
```

Comment:
Maybe it's not easy to keep order consistency some days later, how about use a function to get name of type?

Reply:
OK